tests/console/commands/test_check.py

```diff
@@ -254,3 +306,25 @@ def test_check_lock_up_to_date(
 
     # exit with an error
     assert status_code == 0
+
+
+def test_check_does_not_error_on_pypi_reference(
+    command_tester_factory: CommandTesterFactory,
+    poetry_with_pypi_reference: Poetry,
+) -> None:
+    tester = command_tester_factory("check", poetry=poetry_with_pypi_reference)
+    status_code = tester.execute("")
+
+    # Warnings about deprecated syntax are expected
```

Comment:
Shouldn't we just avoid deprecated syntax in this test because it distracts?

Reply:
Yeah, right.